js/store.js

```diff
@@ -192,35 +205,26 @@ const store = {
     }
   },
 
-  getThirdParties(hostname) {
-    if (!hostname) {
-      throw new Error('getThirdParties requires a valid hostname argument');
-    }
-
-    const firstParty = this.getWebsite(hostname);
-    if ('thirdPartyHostnames' in firstParty) {
-      return firstParty.thirdPartyHostnames;
-    }
-
-    return null;
-  },
-
   setWebsite(hostname, data) {
     const websites = clone(this._websites);
```

Comment:
@jonathanKingston Can you try building this PR and looking at the console? `store.addFirstPartyLink` gets called a bunch of times before it finishes writing the `firstParty` object to the database from the very first call... What is happening here, and how can I fix it?